docs/requirements.txt

```diff
@@ -4,4 +4,8 @@ sphinx-markdown-tables==0.0.9
 sphinx-rtd-theme==0.4.2
 sphinxcontrib-websupport==1.1.0
 recommonmark==0.5.0
-nni==0.5
\ No newline at end of file
+hyperopt
+json_tricks
```

Comment:
I wonder this package is for docs or for sdk?

Reply:
those dependencies is added so that Tuners can be successfully installed without installing NNI first.